app/scripts/modules/azure/serverGroup/configure/wizard/CloneServerGroup.azure.controller.js

```diff
@@ -4,6 +4,8 @@ const angular = require('angular');
 
 import { SERVER_GROUP_WRITER, TaskMonitor, ModalWizard, FirewallLabels } from '@spinnaker/core';
 
+const Utility = require('../../../utility').default;
```

Comment:
It's preferable to use `import Utilility from '../../../utility` where possible.  Was there a specific reason not to use that here?